### PR TITLE
Issue #480

### PR DESCRIPTION
**Pull request by prodan-eric**
Added a new prop called cellDisplayLimit (with a default of 10) to handle the max number of visible cells in the matrix

### CODE REVIEW

File: src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue [hunk 1]
```diff
@@ -575,9 +583,8 @@ export default defineComponent({
     grid-template-columns: repeat(var(--matrix-rows), 1fr);
 
     &__cell {
-        font-size: max(calc(var(--cell-dimensions) * 0.1), 12px);
+        font-size: calc(var(--cell-dimensions) * 0.33);
```
Comment (fadiDL) [inline src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue:586]: @prodan-eric have you tested the case of one and two matricies in the demos ? 
does this 0.33 work well ?

File: src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue [hunk 2]
```diff
@@ -277,7 +283,10 @@ export default defineComponent({
         const { variables } = useThemeVariables()
         const currentBrushState = ref<{ min: number; max: number }>({
             min: 0,
-            max: props.matrix.length
+            max:
```
Comment (fadiDL) [inline src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue:286]: id just use Math.min in this case as to add readability

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue (hunk 2 above)
Comment (prodan-eric) [reply src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue:286]: True, I'll change it

File: src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue (hunk 1 above)
Comment (prodan-eric) [reply src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue:583]: Yeah, it's the best font scaling I could come up with. Seems to work well in all cases

**Review by fadiDL — APPROVED** [review]

**Pull request merged**